AI/ai_strategy.py

```diff
@@ -132,12 +137,16 @@ def __init__(self, second_player: bool = False):
         self._ai_uuid = "108eaa05-2b0e-4e00-a190-8856edcd56a5"
         self._ai_uuid2 = "108eaa05-2b0e-4e00-a190-8856edcd56a6"
```

Comment:
generate uuid2 randomly

```suggestion
        self._ai_uuid2 = "d90397a5-a255-4101-9864-694b43ce8a6c"
```